### PR TITLE
Modifying the README.md file.

### PR DESCRIPTION
**Pull request by Zhilei**
From the most recent hands-on experience on the NRAO server.

### CODE REVIEW

**Review by bhazelton — COMMENTED** [review]
This is interesting. It's not how I have previously set up git-lfs and it doesn't really match the directions here: https://git-lfs.github.com, which say of the install command "You only need to run this once per user account." Do you have a sense of whether something has changed in git-lfs or if there's just differences machine to machine? If it's the latter, maybe we want both sets of instructions?

**Comment by Zhilei** [comment]
Aha, that's a really good point.
I have to admit that this repo is my first git-lfs related repo so my account was definitely not set up before this.
I guess now my experience will not be able to distinguish whether it is account-specific or repo-specific.
But I am leaning towards changing the readme.md file to be consistent with the git-lfs website. Should I give it a go?

**Comment by bhazelton** [comment]
Yes, let's stick closer to the git-lfs instructions unless we can be sure that they don't work.

File: README.md [hunk 1]
```diff
@@ -6,4 +6,8 @@ Users who wish to contribute beam models should create a new directory with a de
 # Installation
 The data files in this repo are large and are actually stored in git LFS. To get the actual files (rather than just the small pointers), you'll need to install git LFS following the directions [here](https://help.github.com/en/articles/installing-git-large-file-storage).
 
-If you install git LFS before you clone this repo, the files should be downloaded properly when you clone it normally (`git clone <repo_url>`). If you already have a clone of this repo and the files are just small pointers, you can use `git lfs pull` to get the actual files.
+The repo can be cloned via `git clone <repo_url>`. This operation gives the structure of the repo and pointers to the files.
+
+Then one should run `git lfs install` for the first-use of git lfs for one account.
```
Comment (bhazelton) [inline README.md:11]: Did you try installing git lfs before you cloned the repo? My experience was that if you had already installed git lfs, then when you cloned the repo you got the full data files with the clone. I'm wondering if that's still true. If it is, I think making it clear that you can do that too would be nice. As it reads now, it looks like you have to clone before you install git lfs and I don't think that's required.

Comment (Zhilei) [reply README.md:11]: That's a good point! Let me implement this change!

**Review by bhazelton — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**